ReleaseProcess.md

```diff
@@ -0,0 +1,97 @@
+This is an early draft of the release procecss for icecc.  It will need to be refined over time.
+
+The steps to do a github release are
+
+1. Create a branch for the new release
+  1. update the version in master
+1. Create a release canidate (optional)
+1. Test
+1. run `make distcheck` and fix errors
+1. Create a full release
+  1. Update the version numbers in the release branch
+    1. Push this change
+  1. run `make dist`
+    1. Verify creates packages build in a clean directory
+  1. On Gihub create a new release
+    1. Upload the files create by `make dist`
+    1. Create a new tag for this release
+    1. Add any notes that are particulary interesting
+1. Repeate the release process if needed for minor releases from this branch, or start over with a new release from master
+
+# Updating versions Numbers
+Version numebrs need to be updated several times in this process. 
+
+Master need to always have the version numbers like 1.X.90 where X is the version from the previous release.
+Official releases have versions numbers of 1.X.Y where X is the new release number (minor version) and Y increases as for each "micro" elease.  Typically icecc does not do micro releases, but sometimes a release is done wrong and so a correction release is made, or an important bug is fixed.
+
+In the file `configure.ac` find the lines for icecream_version_minor and icecream_version_micro, update these as needed.  The following example is for version 1.4.90, and so is what you might expect to see in th master branch util release 1.5.0 is made.
+```
+m4_define([icecream_version_minor],[4])
+m4_define([icecream_version_micro],[90])
+```
+Comit the above changes and push to the correct branch.
```

Comment:
Missing m in commit

Reply:
I have dysgraphia which is a fancy way of saying I'm a bad speller